lib/blacklight/configuration.rb

```diff
@@ -156,6 +156,12 @@ def initialized_default_configuration?
       # set to Blacklight::Icons::BookmarkIconComponent to replace checkbox with icon
       property :bookmark_icon_component, default: nil
 
+      # @!attribute full_width_header
+      # @since v8.3.1
+      # When true, doesn't place the header in col-9 documents but outside as own div
```

Comment:
I'm not sure it's clear to say "in col-9 documents".  Can you be more descriptive here?

Is there a reason we need to support this flag? Can we do it just one way?  This sort of configuration makes it challenging to test as it adds a whole new dimension that we should be testing for.  Is it possible for just your application to override the correct components to get the layout as you need?